client-ts/signalr/src/HttpConnection.ts

```diff
@@ -33,13 +38,15 @@ export class HttpConnection implements IConnection {
     private readonly httpClient: HttpClient;
     private readonly logger: ILogger;
     private readonly options: IHttpConnectionOptions;
+    private readonly transferFormat: TransferFormat;
     private transport: ITransport;
     private connectionId: string;
     private startPromise: Promise<void>;
 
     public readonly features: any = {};
 
-    constructor(url: string, options: IHttpConnectionOptions = {}) {
+    constructor(url: string, transferFormat: TransferFormat, options: IHttpConnectionOptions = {}) {
```

Comment:
Should `transferFormat` have validation that it is present, or a default? Just thinking about non-TS users

Reply:
It seems TypeScript even allows random numbers to be passed in here, so validation is definitely a Good Thing.

Reply:
Should transferFormat be passed into Start like it is in C#?